lib/koans/10_map_sets.ex

```diff
@@ -5,19 +5,27 @@ defmodule MapSets do
 
   @set MapSet.new([1, 2, 3, 4, 5])
 
-  koan "I am merely another collection but, you can perform some operations on me" do
-    new_set = MapSet.new(@set, fn x -> 3 * x end)
-
-    assert MapSet.member?(new_set, 15) == ___
-    assert MapSet.member?(new_set, 1) == ___
-  end
+  koan "I am very similar to a list" do
+    assert Enum.fetch(@set, 0) == {:ok, ___}
+  end 
 
   koan "However, I do not allow duplication" do
     new_set = MapSet.new([1, 1, 2, 3, 3, 3])
 
     assert MapSet.size(new_set) == ___
   end
 
+  koan "Even though I am like a list I am unordered after 32 elements" do
+    assert 1..33 |> MapSet.new() |> Enum.fetch(0) == {___, 11}
```

Comment:
Guys, I am doing the koans now and arrived at this exact koan (as you can guess I am new to elixir).

After I did not got any results from searching with various combinations of `elixir mapset 32 limit` I found this pr and your comments.

Can you please elaborate or point me to the elixir (or erlang) docs that explains this @lamvery's does not make sens with my limited knowledge.